src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

```diff
@@ -1283,13 +1323,13 @@ private void UpdateImage(int key, PropertyValue value)
         /// Callback function to Lazy UpdateImage.
         /// Or, We can call UpdateImage() function directly if we need.
         /// </summary>
-        private void UpdateImage(object source, EventArgs e)
+        internal protected void UpdateImage(object source, EventArgs e)
         {
             UpdateImage();
             _imagePropertyUpdateProcessAttachedFlag = false;
         }
 
-        private void UpdateImage()
+        internal protected void UpdateImage()
```

Comment:
Same comment for this method.
Please consider this one more.

Reply:
I hope subclass can call this API. so I'll make it protected internal level.

Reply:
As you said, it can be used by subclass if you change it to protected internal.
But if there is any class inherit it in app side, app can call it.
If it is right you wanted, you need to make it hidden.

Reply:
App can do same jobs what you explained in `ViewInternal.cs` 's `LayoutCount` property. We can make it as public when app side have some bad mind.
In this case, Should we also make that property as hidden? I don't think so.